src/cloudevent/mocks/helpers.ts

```diff
@@ -1,4 +1,5 @@
 import { CloudEvent, CloudFunction } from 'firebase-functions/v2';
+import { Expression } from 'firebase-functions/v2/params';
```

Comment:
Aside - we are planning to change the import path of Expression from `v2/params` to  just `/params`.

I wonder if there are more "elegant' way to deal with this to avoid having this problem 🤔

Reply:
When is the change going to occur? 🙃

Reply:
v4 sdk!